client/me/help/style.scss

```diff
@@ -46,3 +46,8 @@
 	font-size: 14px;
 	color: $gray-dark;
 }
+
+.help-contact-form__help-language-prompt {
+    font-size: 14px;
+    font-weight: 600;
```

Comment:
I was going to suggest using a `<strong>` element instead of a div here, as it already has a font-weight of 600. But it's probably just nitpicking and it seems like `font-weight: 600` is specified manually in many other parts of Calypso